lib/tasks/settings.rake

```diff
@@ -13,4 +13,11 @@ namespace :settings do
     Setting['feature.allow_attached_documents'] = true
   end
 
+  desc "Initialize max size, number, allowed and accepted content types"
```

Comment:
Good, this rake task should initialize all new settings

Reply:
Thanks @voodoorai2000 I will get started with the edits you recommend above. I'm so happy to be contributing to this project! Thanks for all your support!

Reply:
Awesome! Thank you @cheneyshreve 🙏🎉👏